e2e-tests/cypress/integration/manageSettings/manageStandaardTeksten.spec.ts

```diff
@@ -11,6 +11,9 @@ import * as createSignal from '../../support/commandsCreateSignal';
 import * as general from '../../support/commandsGeneral';
 
 describe('Standaardteksten', () => {
+  beforeEach(() => {
+    general.setResolution([1024,768]);
```

Comment:
What's the reason for setting this to a different resolution from the other tests? Can the resolution be extracted to a constants file?